### PR TITLE
DEL: Reset keychain after uninstall

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/d8a0up2hhvvzn34c0qcvtnw7a0

[download apk](http://filestorage.bluewallet.io:1488/1610639448.apk)

File: screen/wallets/walletMigrate.js [hunk 1]
```diff
@@ -15,41 +10,6 @@ export default class WalletMigrate {
 
   // 0: Let's start!
   async start() {
-    if (Platform.OS === 'ios' && !isCatalyst) {
```
Comment (Overtorment) [inline screen/wallets/walletMigrate.js:18]: image

**Review by Overtorment — APPROVED** [review]
lgtm

**Comment by limpbrains** [comment]
How does this work? I see only removing of this feature.

**Comment by marcosrdz** [comment]
> How does this work? I see only removing of this feature.

keychain and app live separately. this allows you to remove the app from your device, but keep the data. sadly, there were multiple bugs after the app grew and started using App Groups (for widgets and watch app) which resulted in this feature being triggered,with wrong value, without the user requesting it.

**Comment by Overtorment** [comment]
@limpbrains so the idea is that this feature randomly wipes keystores of random newb users, and better to just remove it instead of having a headache where the bug is

**Review by limpbrains — APPROVED** [review]

**Review by ncoelho — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**